### PR TITLE
add cert-manager-webhook-pdns

### PR DESCRIPTION
**Pull request by Dentrax**
> **Note**
> **Before:** `71.5MB, 113 packages, 13 CVEs`
> **After:** `57.4MB, 101 packages, 0-CVE`

### Image Size

- [X] The Image is smaller in size than its common public counterpart.
- [ ] The Image is larger in size than its common public counterpart (please explain in the notes).

Notes:

### Image Vulnerabilities

- [X] The Grype vulnerability scan returned 0 CVE(s).
- [ ] The Grype vulnerability scan returned > 0 CVE(s) (please explain in the notes).

Notes:

### Image Tagging

- [ ] The image is _not_ tagged with version tags.
- [X] The image is tagged with :latest
- [ ] The image is not tagged with :latest (please explain in the notes).

Notes:

### Basic Testing - K8s cluster

- [X] The container image was successfully loaded into a kind cluster.
- [ ] The container image could not be loaded into a kind cluster (please explain in the notes).

Notes:

### Basic Testing - Package/Application

- [X] The application is accessible to the user/cluster/etc. after start-up.
- [ ] The application is not accessible to the user/cluster/etc. after start-up. (please explain in the notes).

Notes:

### Helm

- [X] A Helm chart has been provided and the container image can be used with the chart.  If needed, please add a -compat package to close any gaps with the public helm chart.
- [ ] A Helm chart has been provided and the container image is not working with the chart (please explain in the notes).
- [ ] A Helm chart was not provided.

Notes:

### Processor Architectures

- [X] The image was built and tested for x86_64.
- [ ] The image could not be built for x86_64 (please explain in the notes).
- [X] The image was built and tested for aarch64.
- [ ] The image could not be built for aarch64. (please explain in the notes).

Notes:

### Functional Testing + Documentation

- [x] Functional tests have been included and the tests are passing.  All tests have been documnted in the notes section.

Notes: Their Deployment uses Readiness probes so additional functional tests might not need to.

### Environment Testing + Documentation

- [X] There has not been a request and/or there is no indication that this image needs tested on a public cloud provider.
- [ ] The container image has been tested successfully on a public cloud provider (AWS, GCP, Azure).
- [ ] The container image has not been tested successfully on a public cloud provider (AWS, GCP, Azure) (please explain in the notes).

Notes:

### Version

- [X] The package version is the latest version of the package.  The latest tag points to this version.
- [ ] The package version is the not the latest version of the package (please explain in the notes).

Notes:

### Dev Tag Availability

- [X] There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
- [ ] There is not a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base') (please explain in the notes).

Notes:

### Access Control + Authentication

- [X] The image runs as `nonroot` and GID/UID are set to 65532 or upstream default
- [ ] Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
- [ ] The image requires a non-standard username or non-standard GID/UID (please explain in the notes).

### ENTRYPOINT

- [X] applications/servers/utilities set to call main program with no arguments e.g. [redis-server]
- [ ] applications/servers/utilities not set to call main program with no arguments e.g. [redis-server] (please explain in the notes)
- [ ] base images leave empty.
- [ ] base image and not empty (please explain in the notes).
- [ ] dev variants is set to entrypoint script that falls back to system.
- [ ] dev variants is not set to entrypoint script that falls back to system (please explain in the notes).

### CMD

- [ ] For server applications give arguments to start in daemon mode (may be empty)
- [ ] For utilities/tooling bring up help e.g. `–help`
- [ ] For base images with a shell, call it e.g. [/bin/sh]

### Environment Variables

- [X] Environment variables added.
- [ ] Environment variables not added and not required.

### SIGTERM

- [X] The image responds to SIGTERM (e.g., `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`)

### Logs

- [X] Error logs write to stderr and normal logs to stdout. Logs DO NOT write to file.

### Documentation - README

- [X] A README file has been provided and it follows the README template.

### CODE REVIEW

File: images/cert-manager-webhook-pdns/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<!--body:start-->
+## Usage
+
+This image is a drop-in replacement for the upstream image.
+You can run it using the Helm Chart with:
+
+```shell
+$ helm repo add cert-manager-webhook-pdns https://zachomedia.github.io/cert-manager-webhook-pdns
+$ helm install cert-manager-webhook-pdns cert-manager-webhook-pdns/cert-manager-webhook-pdns \
+    --set image.registry=gr.dev/chainguard/cert-manager-webhook-pdns \
```
Comment (mamccorm) [inline images/cert-manager-webhook-pdns/README.md:37]:
```suggestion
    --set image.registry=cgr.dev/chainguard/cert-manager-webhook-pdns \
```

File: images/cert-manager-webhook-pdns/tests/main.tf [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    {
+      name = "Helm install"
+      cmd  = module.helm.install_cmd
+    }
+  ]
+
+  labels = {
+    type = "k8s",
+  }
+}
```
Comment (mamccorm) [inline images/cert-manager-webhook-pdns/tests/main.tf:78]: Can we add a shell test to validate functionality? i.e after installing into a cluster, add [powerDNS helm chart](https://artifacthub.io/packages/helm/halkeye/powerdns) and a test for this image?

Or if theres a technical blocker from doing that, a TESTING.md file with manual validation steps?

Comment (Dentrax) [reply images/cert-manager-webhook-pdns/tests/main.tf:78]: Actually I did trust their Readiness probes but I'll dig into this see what can we do here. That Helm Chart seems a bit oldie but will check. No blockers. 👍

Comment (mamccorm) [reply images/cert-manager-webhook-pdns/tests/main.tf:78]: Yea good point, or maybe just spin up the container if the chart is old. But if you think it's not feasible to fully automate some manual steps for validation would be awesome :)

**Pull request merged**